### PR TITLE
Pd 1011 update core to scale migration articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates content in CORE to SCALE migration articles and snippets.
It deletes one snippet only used in one article (not found in a search of it in the repo) and moves that content into the MigratePrep..md article.
It reorganizes and improves the content in the MigratePrep.md article to emphasize the areas users need to address before they migrate/update to or clean install SCALE.

It adds a missing step in the modified MigratingFromCORE.md article.

It adds information about the force option when updating to Dragonfish when deprecated services are in use to the Release Notes and changes the word reinstall to install in the upgrade path mermaid.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 1]
```diff
@@ -8,76 +8,115 @@ tags:
...
+   
+   <input type="checkbox"> CAs, certificates, CSRs - If you added certificate authorities, certificates or certificate signing requests to CORE, these should migrate with the system config file, but as a precaution against possible malformed certificates copy or screenshot all CA, certificate, and CSR settings, and save the public and private keys of each. Make sure you have backed-up copies of certificates used in CORE to import or configure in SCALE.
+   
+   <input type="checkbox"> Usernames beginning with (0-9) - Review local user account names and rename or replace these with a letter or underscore before migrating.
    When you set up the first required pool on SCALE import this pool first.
 
-5. Back up your stored data files.
-   If you need to do a clean install with the SCALE <kbd>iso</kbd> file, you can import your data pools into SCALE.
+   <input type="checkbox"> Tunables on CORE - SCALE does not use **Tunables** the way CORE does. SCALE provides script configuration on the **System Settings > Advanced** screen as **Sysctl** scripts. A future release of SCALE could introduce similar tunables options found in CORE but for now it is not available.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratePrep.md:54]: Don't think we want to promise future functionality that hasn't necessarily been scheduled or confirmed yet.
```suggestion
   <input type="checkbox"> Tunables on CORE - SCALE does not use **Tunables** the way CORE does. SCALE provides script configuration on the **System Settings > Advanced** screen as **Sysctl** scripts.
```

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 2]
```diff
@@ -8,76 +8,115 @@ tags:
...
 
-   * CORE init/shutdown scripts to add to SCALE.
+4. Write down your network configuration information or take screenshots of the global network settings, interfaces, static IP addresses, and aliases configured.
+   
+   FreeBSD and Linux use different nomenclature for network interfaces, bridges, LAGGs, and VLANs. 
+   Because of the difference, network settings can either get lost or not transfer which means you have no network connectivity.
+   You can find interface names in the CORE UI on the **Network > Interfaces** screen.
 
-   * CORE cron jobs configured if you want to set the same jobs up in SCALE.
+   If using TrueNAS System servers, to find the Linux interface ID equivalent, refer to the network port ID manuals for your model found in the [Solutions > TrueNAS Systems](https://www.truenas.com/docs/hardware/) area of the Documentation Hub for network port assignments.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratePrep.md:72]:
```suggestion
   When using a TrueNAS Enterprise system from iXsystems, refer to the network port ID manuals of your [TrueNAS Systems](https://www.truenas.com/docs/hardware/) to find the network port assignments in TrueNAS SCALE.
```

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 3]
```diff
@@ -8,76 +8,115 @@ tags:
...
-   * CORE init/shutdown scripts to add to SCALE.
+4. Write down your network configuration information or take screenshots of the global network settings, interfaces, static IP addresses, and aliases configured.
+   
+   FreeBSD and Linux use different nomenclature for network interfaces, bridges, LAGGs, and VLANs. 
+   Because of the difference, network settings can either get lost or not transfer which means you have no network connectivity.
+   You can find interface names in the CORE UI on the **Network > Interfaces** screen.
 
-   * CORE cron jobs configured if you want to set the same jobs up in SCALE.
+   If using TrueNAS System servers, to find the Linux interface ID equivalent, refer to the network port ID manuals for your model found in the [Solutions > TrueNAS Systems](https://www.truenas.com/docs/hardware/) area of the Documentation Hub for network port assignments.   
+   If not using TrueNAS system hardware, refer to the manual or documentation provided with your system or locate this information on your server hardware and take note of it.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratePrep.md:73]:
```suggestion
   When using custom hardware for TrueNAS, refer to the manual or documentation provided with your system or locate this information on your server hardware and take note of it.
```

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 4]
```diff
@@ -8,76 +8,115 @@ tags:
...
-   * Cloud storage backup provider credentials configured in CORE if you do not have these recorded in other files kept secured outside of CORE.
+5. Migrate the deprecated S3 MinIO service (if in use). See [services deprecated in SCALE](#migrating-from-deprecated-services).
+   Read and follow instructions in [Migrating from MinIO S3](https://www.truenas.com/docs/scale/22.12/scaletutorials/apps/communityapps/minioclustersscale/migratingfroms3service/)!
+   Make sure S3 MinIO data is backed up as a precaution. The S3 service requires installing SCALE 22.12.3 (Bluefin).
+   This version of SCALE provides access to both the S3 service and the MinIO app you migrate to. 
 
-   * Replication, periodic snapshot, cloud sync, or other tasks settings to reconfigure in SCALE if you want to duplicate these tasks.
+6. Back up your stored data files.
+   If you need to do a clean install with the SCALE <file>iso</file> file, you can import your data pools into SCALE.
+   Verify you can access your backed up filed before you upgrade/migrate to SCALE.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratePrep.md:85]:
```suggestion
   Verify you can access your backed up files before you upgrade/migrate to SCALE.
```

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 5]
```diff
@@ -8,76 +8,115 @@ tags:
...
 
-   * Make sure you have backed-up copies of certificates used in CORE to import or configure in SCALE.
+7. Back up any critical data.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratePrep.md:87]: 6. and 7. are essentially the same thing, go ahead and combine.

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 6]
```diff
@@ -67,6 +65,10 @@ If this process fails, retry using the iso file method above.
 
    ![SCALESidegradeReboot](/images/SCALE/SystemSettings/SidegradeRestart.png  "Reboot to Finish")
 
-8. When TrueNAS SCALE boots, you might need to [use the Shell to configure networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
+When TrueNAS SCALE boots, you might need to [use the Shell to configure the primary networking interfaces]({{< relref "ConsoleSetupMenuScale.md" >}}) to enable GUI accessibility.
+
+After booting and gaining access to the UI, go to **System Settings > General** and [upload the system config file](/scale/scaletutorials/systemsettings/general/managesysconfigscale/#uploading-the-file). This migrates your CORE settings, imports your pools, shares, etc. into SCALE.
+
+After uploading the config file, review each area of the UI previously configured in CORE to validate pools imported and settings migrated correctly. Begin with your network settings.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:73]:
```suggestion
We'll want to add a step here to create a new SCALE built-in administrator account, as it appears that uploading the CORE config file destroys the admin user created during the SCALE install process (tested with 13.0-U6.1 > 24.04-nightly build).
```

File: content/SCALE/GettingStarted/SCALEReleaseNotes.md [hunk 7]
```diff
@@ -104,6 +104,10 @@ After install, restore the system configuration and import the pools.
 Depending on the specific system configuration, this can be a straightforward or complicated process.
 See the [Migration articles]({{< relref "/SCALE/GettingStarted/Migrate/_index.md" >}}) for cautions and notes about differences between each software and the CORE to SCALE migration process.
 
+The option to change the Train to SCALE 24.04 (Dragonfish) no longer exists as a migration option.
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/SCALEReleaseNotes.md:107]:
```suggestion

```

File: content/_includes/RootToAdminUserAccount.md [hunk 8]
```diff
@@ -1,14 +1,21 @@
 &NewLine;
 
 {{< hint type=important title="SCALE Administrator Accounts" >}}
-Root is the system administration account for CORE and the Angelfish and early Bluefin releases of SCALE.
-Users migrating from CORE to SCALE or from the pre 22.12.3 release of SCALE Bluefin must manually create an admin user account.
+Root is the system administration account for CORE, SCALE Angelfish, and early Bluefin releases.
```
Comment (Mrt134) [inline content/_includes/RootToAdminUserAccount.md:4]:
```suggestion
Root is the default system administration account for CORE, SCALE Angelfish, and early Bluefin releases.
```

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 9]
```diff
@@ -8,76 +8,115 @@ tags:
...
 
 {{< include file="/_includes/COREMigratesList.md" >}}
 
 ## Preparing for Migration
-
-Before you attempt to migrate your CORE system to a SCALE major version:
-
-1. Upgrade your CORE system to the most recent publicly-available CORE version.
+Before you attempt to migrate your CORE system to a SCALE major version. 
+{{< hint type="warning" title="Using USB Devices" >}}
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratePrep.md:20]: This was in the context as the medium for backups. I've clarified this content in the next commit that has suggested PR changes.

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md [hunk 10]
```diff
@@ -8,76 +8,115 @@ tags:
...
+{{< /hint >}}
+{{< enterprise >}}
+CORE Enterprise customers are encouraged to contact Support for assistance with the process of moving from CORE to SCALE, especially customers with HA systems.
+{{< /enterprise >}}
+
+1. Upgrade your CORE system to the most recent publicly-available CORE major maintenace release version.
    TrueNAS systems on 12.0x or earlier should upgrade to the latest CORE 13.0 release (e.g 13.0-U6.1 or newer) prior to migrating to SCALE.
    CORE systems at the latest 13.0 release can use the [iso upgrade](#migrating-using-an-iso-file-to-upgrade) method to migrate to SCALE.
 
+   Migrate [GELI-encrypted pools](https://www.truenas.com/docs/core/coretutorials/storage/pools/storageencryption/#geli-pool-migrations) to a non-GELI-encrypted pool before upgrading from CORE 12.0x or earlier releases!
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratePrep.md:33]: Added as step 2, then renumbered the rest of the list.

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratePrep.md:54]: This was in the article already. I'll reword this.

File: content/SCALE/GettingStarted/Migrate/MigratePrep.md (hunk 5 above)
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratePrep.md:87]: Combined in the next commit.

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md (hunk 6 above)
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:73]: I'll get a snippet created for this to avoid duplicate instructions.

Comment (micjohnson777) [reply content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:77]: Added a new snippet in the next commit.

**Review by Mrt134 — APPROVED** [review]
Additional local build test and review looks good. We'll need to address the scale CLI text at a future point.

**Pull request merged**